app/models/pregnancy.rb

```diff
@@ -1,11 +1,17 @@
-class Pregnancy
+class Pregnancy<ActiveRecord::Base
+  include Auditable
   include Mongoid::Document
   include Mongoid::Timestamps
   include Mongoid::History::Trackable
   include Mongoid::Userstamp
+<<<<<<< HEAD
```

Comment:
Can you remove lines 7-10 here? Make sure you still have `include LastMenstrualPeriodHelper`.